### PR TITLE
Refactor/styles home header

### LINKED ISSUE #124: Block Explorer UI update
- Relocate SearchSelect component for these views:
1. home-main:
 https://www.figma.com/file/GS7thHuVvP72ZrnBkWrVAp/block-explorer?node-id=514%3A883&t=HJIemTQrQSXRDN8V-4

2. home_collapse_hero:
https://www.figma.com/file/GS7thHuVvP72ZrnBkWrVAp/block-explorer?node-id=520%3A1097&t=HJIemTQrQSXRDN8V-4

### PR DESCRIPTION
**Pull request by Thistle-Strum**
Screen Shot 2023-01-17 at 6 29 45 PM
Screen Shot 2023-01-17 at 6 29 58 PM

🔥 Summary
-Updated UI as specified at
    home-main:
    https://www.figma.com/file/GS7thHuVvP72ZrnBkWrVAp/block-explorer?node-id=514%3A883&t=HJIemTQrQSXRDN8V-4

home_collapse_hero:
https://www.figma.com/file/GS7thHuVvP72ZrnBkWrVAp/block-explorer?node-id=520%3A1097&t=HJIemTQrQSXRDN8V-4

😤 Problem / Goals
-Update UI by reducing styles and removing the HeroHeading for returning visitors.

### CODE REVIEW

File: frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx [hunk 1]
```diff
@@ -27,12 +28,13 @@ export const AccountDetailsCard: React.FC<AccountDetailsCardProps> = ({
   account,
   balance,
 }) => {
-  const [isHashTruncated, setIsHashTruncated] = useState<boolean>(true);
+  const [isTruncated, setIsTruncated] = useState<boolean>(true);
```
Comment (hoffmannjan) [inline frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx:31]: :+1:

File: frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx [hunk 2]
```diff
@@ -43,19 +45,19 @@ export const AccountDetailsCard: React.FC<AccountDetailsCardProps> = ({
...
             ) : (
               <Hash hash={trimmedAccountHash} />
             )}
           </HashHeading>
         </AvatarHashContainer>
       </HeadContentContainer>
-      <HashButton type="button" onClick={toggleHashView}>
-        {isHashTruncated ? 'Expand' : 'Collapse'}
+      <HashButton type="button" onClick={toggleHashView} isMobile={isMobile}>
+        {isTruncated ? 'Expand' : 'Collapse'}
```
Comment (hoffmannjan) [inline frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: probably we need to wrap it into translations component, could you look @valfaro123 ?

**Review by hoffmannjan — APPROVED** [review]
LGTM, only a few translations issues that I found :)

File: frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx (hunk 2 above)
Comment (Thistle-Strum) [reply frontend/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx]: I just updated the translation object for the expand/collapse buttons for DeployDetails, AccountDetails, and BlockDetail.

File: frontend/src/components/layout/LogoComponents/DefaultHeaderLogo.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  }
+
+  @media (min-width: ${breakpoints.lg}) {
+    padding: 0;
+  } ;
+`;
+
+export const BlueCasperLogo = styled(BlueLogo)`
+  width: 10%;
+  margin-right: 5px;
```
Comment (valfaro123) [inline frontend/src/components/layout/LogoComponents/DefaultHeaderLogo.tsx:42]: we can change this to rem to be consistent

**Pull request merged**